File: 2577.py
a = int(input())
b = int(input())
c = int(input())
answer = a*b*c
answer = str(answer)
arr = [0,0,0,0,0,0,0,0,0,0]
for i in range(len(answer)):
    for j in range(11):
        if j == int(answer[i]):
            arr[j] += 1
for k in range(len(arr)):
    print(arr[k])

File: 2577.c
#include <stdio.h>
int main(void){
    int a,b,c;
    int arr[10] = {0};
    scanf("%d%d%d",&a,&b,&c);
    int answer = a*b*c;
    while(answer != 0){
        arr[answer%10] += 1;
        answer /= 10;
    }
    for(int i = 0; i < 10; i++){
        printf("%d\n",arr[i]);
    }
}